### DISCUSSION TITLE
v1.0.2

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#2370] [Announcements]
## Windows App SDK 1.0.2 is now available

Windows App SDK 1.0.2 is a stable servicing release that includes critical bug fixes for the 1.0 release. 

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).
- If you are already using the Windows App SDK 1.0 and would like to update to the latest release:

    - Update your **Microsoft.WindowsAppSDK** NuGet package reference from **1.0.0**/**1.0.1** to **1.0.2**.
    - [Download the 1.0.2 runtime](https://aka.ms/windowsappsdk/downloads) using either the standalone Runtime installer (**.exe**) or the Windows App Runtime Redistributable which includes all MSIX package dependencies.

### Bugs fixed in Windows App SDK 1.0.2:

- Fixed layout cycle issue causing an app to crash when scrolling to the end of a ListView: [#6218](https://github.com/microsoft/microsoft-ui-xaml/issues/6218)
- Fixed issue causing C# apps to crash on launch when the C/C++ Runtime (CRT) isn't installed. However, the CRT is still required for C# apps using WebView2: #2117
- Fixed issue where applications with Single-project MSIX did not generate a .appinstaller file: #1821
- Fixed issue where WinUI applications did not support .NET 6 `dotnet build`.

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.0.2'>v1.0.2</a>.</em>

**Comment by Marv51** [comment]
Can you please also update the [productboard](https://portal.productboard.com/winappsdk) with the fixed issues? Single-Project MSIX Appinstaller and multiple windows (from 1.0.1) should be marked as completed there.
